MobileSecondPage.html

```diff
@@ -0,0 +1,107 @@
+</head>
+<body>
+    <span class="upperhead">
+        <div>
+            <img src="./images/Rectangle.png" alt="">
+            <img src="./images/twitter.png" alt="">
+            <a href="#">English</a>
+            <a href="#">MyPage</a>
+            <a href="#">Logout</a>
+        </div>
+    </span>
+    <header>
+        <nav>
+            <div class="burger">
+                <img src="./images/mobile_menu 1.png" alt="" />
+            </div>
+                <div class="header-menu">
+                <span class="close">X</span>
+                <a class="links" id="about" href="./MobileSecondPage.html">About</a>
+                <a class="links" id="myProgram" href="#myProgram">Program</a>
+                <a class="links" id="join" href="#join">Join</a>
+                <a class="links" id="sponsor" href="#sponsor">Sponsor</a>
+                <a class="links" id="news" href="#news">news</a>
+                <a href="#" id="DeskIcon">Worldcup</a>
+            </div>
+        </nav>
+        <div class="logo">
+        <a href="./index.html"><img src="./images/Screenshot 2023-04-new .jpg" alt="world-cup"></a>
+        </div>
+    </header>
+    <section class="headline">
+        <div class="first-headline">
+            <P>"Hello! sharing world"</P>
+            <h1>RUNNING WORLD CUP 2015</h1>
+            <hr>
+        </div>
+        <p class="main-content">Not everyone gets the opportunity to represent their country but in this one you do.
+             No matter your speed or distance, the aim of the game is to run as much as possible for your country to help them reach the top of the leaderboard. The more runs you do; the more km points your country scores.
+             There are 3 knockout rounds before you can be crowned a running world cup champion.</p>
+        <div class="contact-us">
+            <p>Please contact us per Email for nay further questions about running world cup 2015!</p>
+            <a href="#">worldcup2015@cckorea.org</a> 
+        </div>
+        <div class="Subcontent">
+            <h3>running world cup 2015 logo</h3>
+            <hr>
+            <p>The world cup 2015 was decided through the logo competition from 8, June to 7, July.</p>
+            <p id="desktop-content">The logo of mrathon 2015 was decided through the logo competition from 8, June to 7, July.
+                After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+                After consultation with Mr. Agrawal</p>
+            <div>
+                <img src="./images/Screenshot 2023-04-new .jpg" alt="logo">
+            </div>
+        </div>
+        <div class="last-content">
+            <h3>See the past runnig world cup</h3>
+            <hr>
+            <p>Take a look at the past two world cup which took place in Buenos aruond the world</p>
+            <div class="content-img">
+                <img src="./images/how-to-run-faster-1662036469.jpg" alt="">
```

Comment:
As I mentioned above ☝🏽 for the navbar, you should make sure that all of your `img` tags have appropriate `alt` attributes, this will ensure that people can know what the image 🖼️  is about in case it can't loads. ❌